### ISSUE TITLE
ViewBackward not had atrribute 'old_size'

### ISSUE THREAD

**Issue by erickingxu** [starimeL/PytorchConverter#24]
when net output is a list , list can not support out.grad_fn when run FindMultiTops(out.grad_fn),
so I try loop this list like this for making grad_fn working:
 for out in outputs:
        #outputs is my net output, it is a mutiple-list 
        for o in out:
            FindMultiTops(o.grad_fn)
and then, I got this error at follow function:
def flatten(pytorch_layer):
    """ Only support flatten view """
    total = 1
    for dim in pytorch_layer.old_size: # error is here: no attribute 'olde_size'
        total *= dim
 and I run pytorch 0.4.0, so can i make sure it is caused by pytorch version ?

**Comment by starimeL** [comment]
Yes, the errors of attribute not found seemingly are almost caused by higher version. Please make sure you are converting your model on pytorch 0.2.0 (While you can train your model on 0.4.0)
